kola-denylist.yaml

```
@@ -5,17 +5,6 @@
   tracker: https://github.com/coreos/coreos-assembler/pull/1478
 - pattern: podman.workflow
   tracker: https://github.com/coreos/coreos-assembler/pull/1478
-- pattern: ext.config.kdump.crash
-  tracker: https://github.com/coreos/coreos-assembler/issues/2725#issuecomment-1292616121
-  snooze: 2023-02-04
-  arches:
-    - ppc64le
-  streams:
```

We are not building ppc64le for some streams, as the fix probably wasn't promoted to other streams, let's  keep it as:
```suggestion
streams:
     next-devel
     next
     testing
     stable
  ```

---

if @Shilpi-Das1 tested this against `testing-devel` (Fedora 37) and it worked then we can delete all the entries (i.e. we can merge this PR like it is now).

---

Just so it is clear.. we are already running this test for `aarch64`, `x86_64`, and `s390x` everywhere, now we will for `ppc64le` too!

---

A good. I wasn't sure if it had landed in all streams. No changes needed in this case, just ignore my comments above